src/Microsoft.AspNet.Mvc.Razor.Host/Directives/DefaultCodeTreeCache.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.FileProviders;
+using Microsoft.AspNet.Razor.Generator.Compiler;
+using Microsoft.Framework.Cache.Memory;
+
+namespace Microsoft.AspNet.Mvc.Razor.Directives
+{
+    /// <summary>
+    /// Default implementation of <see cref="ICodeTreeCache"/>.
+    /// </summary>
+    public class DefaultCodeTreeCache : ICodeTreeCache
+    {
+        private static readonly MemoryCacheOptions _memoryCacheOptions = new MemoryCacheOptions
```

Comment:
`private static readonly` = PascalCase everywhere else, same comment for `_slidingExpirationDuration`